test/test_optim.py

```diff
@@ -1629,6 +1632,50 @@ def closure():
             res2 = optim_neg_inf.step(closure)
             self.assertEqual(type(res1), type(res2))
 
+    @onlyCPU
```

Comment:
```suggestion
    @onlyCUDA
```

Reply:
Ah...actually this test probably just never ran because it's skipped on CUDA but also requires CUDA...do you mind if I pull this change and add more on top of it so I could test it locally?

Reply:
Oh, Sorry. What I thought was since `TEST_CUDA = torch.cuda.is_available()`, then if we run `test_fused_cpu_matches_cuda(device="cpu") on some CI machine and this CI machine also is `torch.cuda.is_available()`, we will be able to run this test.  But maybe this assumption "some CI machine and this CI machine also is `torch.cuda.is_available()`" is wrong. And meanwhile one `torch.cuda.is_available()` machine will always have `CPU` so your are correct.